### PR TITLE
Add cards to Portfolio Project

### PR DESCRIPTION
**Pull request by Hamdullah-Andar**
### Cards and About sections

- Add the Cards section to Portfolio
- Add the About section to Portfolio
- Use Flex for Designing the layout
- Use Grid for the Designing layout

## Thank you for the review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,6 +37,148 @@ <h1 class="hero">I’m Hamdullah
                 <li class="social-links"><a href="https://vectorstock.com"><img src="images/vector.svg" alt="vector"></a></li>
             </ul>
         </section>
+
+        <section class="projects">
+            <section class="portfolio-projects card1">
```
Comment (NduatiKagiri) [inline index.html]: - [x] Please note that you should only have one `section` element wrapping the whole Works section. You can use `div` for other divisions inside the section. This ensures good/smooth UX/UI.

File: index.html [hunk 2]
```diff
@@ -37,6 +37,148 @@ <h1 class="hero">I’m Hamdullah
...
+
+                <div class="languages-part">
+                    <ul class="all-skills">
+                        <li class="skills">
+                            <h3 class="about-languages">Languages</h3>
+                            <img src="images/down-arrow.svg" alt="">
+                        </li>
+                    </ul>
+
+                    <section class="all-languages">
```
Comment (NduatiKagiri) [inline index.html]: - [x] Please note that you should have only one `section` element wrapping the whole About section. You can use `div` for other divisions inside the section. This ensures good/smooth UX/UI.

File: index.html [hunk 3]
```diff
@@ -37,6 +37,148 @@ <h1 class="hero">I’m Hamdullah
...
+                            <p>Frameworks</p>
+                            <img src="images/larger.svg" alt="">
+                        </li>
+                        <img src="images/Separator-Bottom.svg" alt="">
+                        <li class="skills">
+                            <p>Skills</p>
+                            <img src="images/larger.svg" alt="">
+                        </li>
+                        <img src="images/Separator-Bottom.svg" alt="">
+                    </ul>
```
Comment (NduatiKagiri) [inline index.html]: - [x] All the 3 areas, Languages, Frameworks, and Skills should be list items(`li`s) in one list(`ul`). Please cut the Languages part into this list(`ul`), wrapped in a `li` element.

File: index.html (hunk 2 above)
Comment (NduatiKagiri) [inline index.html]: - [x] Kindly align the languages with the title as aligned in the Figma design.

-   Your design
  Hami1

-   Figma design
  Hami2

-  Please note that the languages have a slightly bigger margin than the title. Check the specified margins and padding from the design and apply them accordingly.

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @Hamdullah-Andar 👋 ,

# CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors
✅ Good PR description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -37,6 +37,144 @@ <h1 class="hero">I’m Hamdullah
...
+                    <img class="counter2" src="images/Counter.svg" alt="">
+                    <li class="year">2015</li>
+                </ul>
+                <p class="card-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li class="html languages-text">html</li>
+                    <li class="css languages-text">css</li>
+                    <li class="javascript languages-text">javaScript</li>
+                </ul>
+                <button class="project-btn project-btn-text">See Project</button>
```
Comment (Kamba56) [inline index.html:78]: - [Optional] Consider adding a hover and pressed state to your buttons to match design template. You can use the `:hover` and `:active` pseudo-classes.
btn1

**Review by Kamba56 — APPROVED** [review]
Hi @Hamdullah-Andar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**